tests/find.spec.ts

```diff
@@ -45,6 +45,32 @@ describe('find', () => {
   })
 })
 
+describe('get', () => {
+  test('returns the element if it exists', () => {
```

Comment:
nice job adding tests, can we put this in a `get.spec.ts` (each wrapper method has it's own file)